salt/python/extensions.sls

```diff
@@ -3,3 +3,10 @@ python c extensions:
     - pkgs:
       - python3-dev
       - build-essential
+
+{% if grains.osmajorrelease|string == '22' %}
```

Comment:
Why do we need `|string` here? (One other occurrence)

Reply:
I was doing this because `grains.osmajorrelease` is an integer and it was erroring in the comparison.
I've just committed switching both values to integers.